editor/project.clj

```diff
@@ -233,7 +238,6 @@
                                                     [org.openjfx/javafx-fxml "22-ea+11"]
                                                     [org.openjfx/javafx-swing "22-ea+11"]
                                                     [com.clojure-goes-fast/clj-async-profiler "0.5.1"]
-                                                    [com.clojure-goes-fast/clj-memory-meter "0.1.2"]
```

Comment:
The `:jamm` profile now has a direct dependency to the underlying `JAMM` library `clj-memory-meter` was using.